### DISCUSSION TITLE
Generating OpenSCAP Reports in JSON Format

### DISCUSSION THREAD

**Discussion by samirMhn** [OpenSCAP/openscap#2133] [General]
Hi,

I am currently using OpenSCAP to generate security reports. These reports are generated in XML and html  format. Is it possible to generate the reports in JSON format instead? If so, could you please provide guidance on how to configure OpenSCAP to output reports in JSON?

Thank you!

**Comment by evgenyz** [comment]
There is no way to do that using only OpenSCAP, but you can try https://github.com/OpenSCAP/openscap-report.
